Development instructions don't work as written

When I check out the repository and attempt to run the dev server according to the instructions, I get an error that `babel-loader` isn't found.

```
$ env --chdir=frontend nix develop -c yarn dev
path '/home/tvb/projects/nixos-search/frontend' does not contain a 'flake.nix', searching up
warning: ignoring untrusted substituter 'https://nixos-search.cachix.org', you are not a trusted user.
Run `man nix.conf` for more information on the `substituters` configuration option.
warning: ignoring untrusted substituter 'https://nixos-search.cachix.org', you are not a trusted user.
Run `man nix.conf` for more information on the `substituters` configuration option.
yarn run v1.22.22
$ webpack serve --port 3000 --config config/webpack.dev.js
<i> [webpack-dev-server] [HPM] Proxy created: /backend  -> https://nixos-search-7-1733963800.us-east-1.bonsaisearch.net/
<i> [webpack-dev-server] [HPM] Proxy rewrite rule created: "^/backend" ~> ""
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:3000/
<i> [webpack-dev-server] On Your Network (IPv4): http://192.168.1.185:3000/
<i> [webpack-dev-server] Content not from webpack is served from '/home/tvb/projects/nixos-search/frontend/src/assets' directory
<i> [webpack-dev-server] 404s will fallback to '/index.html'
ERROR in main
Module not found: Error: Can't resolve 'babel-loader' in '/home/tvb/projects/nixos-search/frontend'

webpack compiled with 1 error
```

I still get the error if I `cd frontend` and `nix develop -c yarn dev`, if I enter the devshell and `yarn dev` manually, or if I `direnv allow` and run `yarn dev` that way.

I am using Nix 2.18.4.

---

While inside the devshell, I can run `yarn install` to generate a `yarn.lock` and `node_modules`, and `yarn dev` works after this. I assume this `yarn install` is intended, so the README can probably be updated to include that. Should the yarn.lock be committed to git? If not, should a .gitignore rule be added to make that decision explicit?

--- a/README.md
+++ b/README.md
@@ -41,6 +41,12 @@ better queries which in turn will produce better results.
 To start developing open a terminal and run:
 
 ```
+env --chdir=frontend nix develop -c yarn install
+```
+
+... and then:
+
+```
 env --chdir=frontend nix develop -c yarn dev
 ```
 

